packages/site-server/src/templates/base.ts

```diff
@@ -58,6 +60,17 @@ export function* renderPage(data: {
   } else {
     yield* data.content;
   }
+
+  if (data.initialData !== undefined) {
+    yield `<script>window.__ssrData = ${JSON.stringify(
+      data.initialData
+    )}</script>`;
```

Comment:
I think it's good practice to sanitize from HTML injection this here (this is what Redux [suggests](https://redux.js.org/usage/server-rendering#inject-initial-component-html-and-state) in their docs):
```suggestion
    ).replace(/</g, '\\u003c')}</script>`;
```

Reply:
thanks!